components/toast/toast.jsx

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {FormattedMessage} from 'react-intl';
+import {OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+import UnreadBelowIcon from 'components/widgets/icons/unread_below_icon';
+import CloseIcon from 'components/widgets/icons/close_icon';
+import Constants from 'utils/constants';
+
+const MIN_TOAST_ZINDEX = 1;
+
+export default class Toast extends React.PureComponent {
+    static propTypes = {
+        onClick: PropTypes.func,
+        onClickMessage: PropTypes.string,
+        onDismiss: PropTypes.func,
+        children: PropTypes.element,
+        show: PropTypes.bool.isRequired,
+        showActions: PropTypes.bool, //used for showing jump actions
```

Comment:
can't we infer this from `onClickMessage` not being empty? or `onClick` not being null (although i can see someone passing a noop function, so maybe the message is better.

Reply:
This is used for hiding the actions of `jump` when at bottom and the toast is present(can be for few secs)